engine/Default/galactic_post_past.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+require_once(get_file_loc('menu.inc'));
+create_galactic_post_menu();
+$template->assign('PageTopic', 'Past <i>Galactic Post</i> Editions');
+
+// View past editions of current game by default
+if (isset($_POST['game_id'])) {
+	SmrSession::updateVar('game_id', $_POST['game_id']);
+} else if (!isset($var['game_id'])) {
+	SmrSession::updateVar('game_id', $player->getGameID());
+}
+$template->assign('SelectedGame', $var['game_id']);
+
+// Get the list of games with published papers
+// Add the current game to this list no matter what
+$db->query('SELECT game_name, game_id FROM game WHERE game_id IN (SELECT DISTINCT(game_id) FROM galactic_post_paper WHERE online_since IS NOT NULL) OR game_id=' . $db->escapeNumber($player->getGameID()) . ' ORDER BY game_id DESC');
+$publishedGames = array();
+while ($db->nextRecord()) {
+	$publishedGames[] = array('game_name' => $db->getField('game_name'),
```

Comment:
It would be nicer to just use `'name'` and `'id'` here, because the game part will be referenced in the var name, eg `$game['id']` and `$game['name']`

Reply:
I agree that if you're looking at the `engine` file, your suggested var names are an improvement because they're less redundant. However, in the `template` file you can't immediately check the query to see what `name` and `id` refer to. You can guess from the name of the variable, but you could also wonder if it is referring to a paper's name and ID (since this code intermingles game and paper queries). I think that hint of doubt warrants the more verbose names.

That said, if you strongly encourage me to change it, I will!